script/lib/install-script-dependencies.js

```diff
@@ -4,6 +4,7 @@ const childProcess = require('child_process');
 
 const CONFIG = require('../config');
 
+// For the 'electron-mksnapshot' dependency
```

Comment:
> // For the `electron-mksnapshot` dependency

This is also for `electron-chromedriver`.

Reply:
@Aerijo could you add `electron-chromedriver` to the comment.

Reply:
I've added it, but I couldn't find any direct reference to Chromedriver using it (then again, I couldn't find their complete changelogs / source either). The closest I've found is https://github.com/electron/chromedriver/pull/52#issuecomment-630845509.

Reply:
It's briefly mentioned in the `README.md`, and I've read the source code and found that it is indeed used.

https://github.com/electron/chromedriver#overriding-the-version-downloaded

`electron-mksnapshot` and `electron-chromedriver` use `@electron/get`.
https://github.com/electron/chromedriver/blob/3d0e75f88efe5e7026ef47e0c6997e25196ea14d/download-chromedriver.js#L3-L8

`@electron/get` respects `ELECTRON_CUSTOM_VERSION`.

- https://github.com/electron/chromedriver/blob/3d0e75f88efe5e7026ef47e0c6997e25196ea14d/download-chromedriver.js#L3-L8
- https://github.com/electron/get/blob/7b295f33d20cfafe556cdb81c56b3e6d006a9ea6/README.md#overriding-the-version-downloaded

This was all tested/confirmed during the electron 6 PR here at the main Atom repo. (Was merged separately from Electron 6). https://github.com/atom/atom/pull/20879

Reply:
Ah thanks, I thought they were talking about the Chromedriver project itself in that comment. I didn't think to check `@electron/get`. I've updated the comment to reflect what is actually recognising that variable, which should help guard against it becoming out of date.